hack/lib/util.sh

```diff
@@ -635,7 +648,8 @@ EOF
 
 # Determines if docker can be run, failures may simply require that the user be added to the docker group.
 function kube::util::ensure_docker_daemon_connectivity {
-  DOCKER=(docker ${DOCKER_OPTS})
+  IFS=" " read -ra DOCKER <<< "${DOCKER_OPTS}"
```

Comment:
It's fail on osx with this error
```
/Users/yago/go/src/k8s.io/kubernetes/hack/lib/util.sh: line 652: DOCKER[@]: unbound variable
```

Reply:
I tried this with bash 3 but can't reproduce...

@yagonobre , if you apply this patch, does it still fail:
```patch
diff --git a/hack/lib/util.sh b/hack/lib/util.sh
index c820206d8e..cef7c94370 100755
--- a/hack/lib/util.sh
+++ b/hack/lib/util.sh
@@ -648,6 +648,7 @@ EOF

 # Determines if docker can be run, failures may simply require that the user be added to the docker group.
 function kube::util::ensure_docker_daemon_connectivity {
+  declare -a DOCKER=()
   IFS=" " read -ra DOCKER <<< "${DOCKER_OPTS}"
   DOCKER=(docker "${DOCKER[@]}")
   if ! "${DOCKER[@]}" info > /dev/null 2>&1 ; then
```

Reply:
Hey, I got it reproduced. A fix (tested on self-compiled bash 3, not on OS X) is available here: https://github.com/kubernetes/kubernetes/pull/74970